dials.go

```diff
@@ -24,6 +24,15 @@ type Params struct {
 	//  - a Verify() method fails after re-stacking when a new version is
 	//    provided by a watching source
 	OnWatchedError WatchedErrorHandler
+
+	// SkipInitialVerification skips the initial call to `Verify()` on any
+	// configurations that implement the `VerifiedConfig` interface.  In cases
+	// where Watching sources are depended upon to provide values that will
+	// successfully verify on the very first access, one should set this to
+	// `true`.  `sourcewrap.Blank` is one such example where values are not
```

Comment:
I think this should be its own paragraph.
The first sentence of the comment is the important description of this field's functionality. The rest is a description of the use-case, which probably needs an empty line separating it from the main description.

Also, this segment is a bit confusing: `In cases where Watching sources are depended upon to provide values that will successfully verify on the very first access`.

Maybe it should be something closer to `In cases where later updates from Watching sources are depended upon to provide a configuration that will be allowed by Verify(), one should set this to true`

Reply:
I like your wording better -- and I've separated the explanation from the description.